src/Resources/skeleton/doctrine/Repository-use-composition.tpl.php

```diff
@@ -0,0 +1,96 @@
+<?= "<?php\n"; ?>
+
+namespace <?= $namespace; ?>;
+
+use <?= $entity_full_class_name; ?>;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+<?= $with_password_upgrade ? "use Symfony\Component\Security\Core\Exception\UnsupportedUserException;\n" : '' ?>
+<?= ($with_password_upgrade && str_contains($password_upgrade_user_interface->getFullName(), 'Password')) ? sprintf("use %s;\n", $password_upgrade_user_interface->getFullName()) : null ?>
+<?= $with_password_upgrade ? "use Symfony\Component\Security\Core\User\PasswordUpgraderInterface;\n" : '' ?>
+<?= ($with_password_upgrade && str_contains($password_upgrade_user_interface->getFullName(), '\UserInterface')) ? sprintf("use %s;\n", $password_upgrade_user_interface->getFullName()) : null ?>
+
+final class <?= $class_name; ?><?= $with_password_upgrade ? " implements PasswordUpgraderInterface\n" : "\n" ?>
+{
+    private EntityRepository $repository;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->repository = $entityManager->getRepository(<?= $entity_class_name; ?>::class);
+    }
+
+    public function find($id, $lockMode = null, $lockVersion = null): ?<?= $entity_class_name; ?><?= "\n"?>
```

Comment:
we should use argument type hints here where possible. Especially if the repo does not implement any sort of interface for these methods.

Reply:
I will do the change.

Reply:
The `ObjectRepository` interface doesn't have any type hint, so I will leave it like this.